Lesson16/serkan/script.js

```diff
@@ -0,0 +1,186 @@
+
+function votingAge(age, student) {
+	if (age < 18 && student) {
+		return 'You are a student and not old enough to vote';
+	} else if ((age >= 18 && age <= 65) && !student ) {
+		return 'You are eligible to vote';
+	} else if (age >= 18 && student) {
+		return 'You are old enough to vote';
+	} else {
+		return 'You are a senior citizen';
+	}
+}
+console.log(checkVotingAge(22, yes));
+
+// 11. Write a function that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of experience are greater than or equal to 10, print "We can offer you the job at the requested salary". (H)
+
+// 12. Write a function that asks the user to enter their score on a test. If the score is less than 60, print "You failed the test". If the score is between 60 and 70 (inclusive), print "You passed the test, but your grade is not great". If the score is between 71 and 80 (inclusive), print "You got a B". If the score is between 81 and 90 (inclusive), print "You got an A". If the score is greater than 90, print "You got an A+ - Great job!". (H)
+
+function testScore(score) {
+	if (score < 60) {
+		return 'You failed the test';
+	} else if (score >= 60 && score <= 70) {
+		return 'You passed the test, but your grade is not great';
+	} else if (score >= 71 && score <= 80) {
+		return 'You got a B';
+	} else if (score >= 81 && score <= 90) {
+		return 'You got a A';
+	} else {
+		return 'You got an A+ - Great job!';
+	}
+}
+console.log(testScore(69));
+
+// 13. Write a function that asks the user to enter a month number (1-12) and displays the corresponding season name. Use a switch statement to handle the different cases. (H)
+
+function getSeasonNames(monthNumber) {
+	switch (monthNumber) {
+		case 1:
+		case 2:
+		case 12:
+			console.log('Winter');
+			break;
+		case 3:
+		case 4:
+		case 5:
+			console.log('Spring');
+			break;
+		case 6:
+		case 7:
+		case 8:
+			console.log('Summer');
+			break;
+		case 9:
+		case 10:
+		case 11:
+			console.log('Fall');
+	}
+}
+console.log(getSeasonNames(7));
+
+// 14. Write a function that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or failing (D, F). Use a switch statement to handle the different cases. (H)
+
+// I'm having a hard time here, to be honest. :)
```

Comment:
This is very much like the exercise above. Take the grade as input (e.g. 
```
function getGrade(grade){
  // switch case goes here, use grade instead of monthNumber 

}

getGrade('C');
```

Case statements would be like this:
```

case 'A':
  console.log('Passed');
	break;


```